src/collective/cover/static/js/main.js

```diff
@@ -1,8 +1,51 @@
 $(function() {
+  // override jquery portlet event
+  // http://stackoverflow.com/a/14063574/2116850
+  function refreshPortlet(hash, _options){
+    var options = {
+        data: {},
+        success: function(){},
+        error: function(){},
+        ajaxOptions: {}};
+    $.extend(options, _options);
+    options.data.portlethash = hash;
+    ajaxOptions = options.ajaxOptions;
+    ajaxOptions.url = $('base').attr('href') + '/@@render-portlet';
+    ajaxOptions.success = function(data){
+      var container = $('[data-portlethash="' + hash + '"]');
+      var portlet = $(data);
+      container.html(portlet);
+      options.success(data, portlet);
+    }
+    ajaxOptions.error = function(){
+      options.error();
+    }
+    ajaxOptions.data = options.data;
+    $.ajax(ajaxOptions);
+  }
+  $('body').undelegate('#calendar-next, #calendar-previous', 'click')
+           .delegate(
+             '.portletWrapper #calendar-next, ' +
+             '.portletWrapper #calendar-previous',
```

Comment:
@hvelarde the problem here is that I am getting the html from the portlet without change, this has the benefit to don't have to maintain this code, but the javascript from this portlet has this problem, the selector to refresh portlet is looking out of the portlet too.  It would be no problem if there were a ``<div>`` with class ``.portletWrapper`` and the portlet id as data attribute to make it work with the original javascript of the portlet, but we don't have it.